src/main/resources/interface-specification.yml

```
@@ -10832,28 +11024,36 @@ components:
       required:
         - id
         - explicitAttributeVerification
-
-    AttributeSeed:
+    CertifiedAttributeSeed:
       title: AttributeSeed
       type: object
       description: Models the attribute registry entry as payload response
       properties:
-        code:
-          type: string
-          description: identifies the unique code of this attribute on the registry
-        kind:
-          $ref: '#/components/schemas/AttributeKind'
         description:
           type: string
+        name:
+          type: string
+        code:
+          type: string
         origin:
+          type: string     
+      required:
+        - description
+        - name
+        - code
+        - origin
```

origin no more required
```suggestion
      required:
        - description
        - name
        - code
```